frontend/src/scenes/surveys/SurveyEditQuestionRow.tsx

```diff
@@ -202,8 +202,16 @@ export function SurveyEditQuestionGroup({ index, question }: { index: number; qu
                     </LemonField>
                 )}
                 {question.type === SurveyQuestionType.Link && (
-                    <LemonField name="link" label="Link" info="Only https:// or mailto: links are supported.">
-                        <LemonInput value={question.link || ''} placeholder="https://posthog.com" />
+                    <LemonField
+                        name="link"
+                        label="Link"
+                        info="Only https:// or mailto: links are supported. Embedding person properties in links requires posthog-js version at least XXX."
```

Comment:
if embedding props is for https links only, we should add a validation to now allow embedding props in mailto URLs, and also mention in the warning message

Reply:
the backend also does some sort of URL validation, we need to make sure the very same checks are being done here and there if we change it.
also we need to make sure that backend wont fail the check with the new embedded prop